src/context.rs

```diff
@@ -1509,7 +1510,7 @@ const PARTITION_CONTEXT_MAX_WIDTH: usize =
 
 const COEFF_CONTEXT_MAX_WIDTH: usize = MAX_TILE_WIDTH / MI_SIZE;
 
-#[derive(Clone)]
+#[derive(Clone, Copy)]
 pub struct BlockContextCheckpoint {
```

Comment:
Looks fairly large, I assume we never pass it around functions.
Is there a clippy lint to prevent it?